spec/spec_helper.rb

```diff
@@ -1,3 +1,5 @@
+require "codeclimate-test-reporter"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.